test/scripts/preconditions/test_preconditions.py

```diff
@@ -107,21 +107,11 @@ def test_empty_required(self):
 
 
 class TestOrgPermissions(unittest.TestCase):
-    def test_base_permissions(self):
```

Comment:
Let's retain this test and assert that the base permissions are empty. This could change again in the future.